ccvm_simulators/solvers/dl_solver.py

```diff
@@ -257,7 +257,10 @@ def _solve(
                 Defaults to None, which generates a filename based on the problem instance name.
 
```

Comment:
Please also remember to remove `post_processor` argument from `_solve()` method and in its docstrings. This applies to all the solvers.

Reply:
Thanks for letting me know this. Nov it is removed from the signature of both `_solve()` and `_solve_adam()`. The docstrings are updated accordingly.